Major Update (#268)
NilPointer-Software/bdfd-wiki: src/bdscript/userPerms.md

```

### Parameters

> - `User ID` `(Type : Snowflake || Flag : Required)` : The user to get permissions for.
> - `Return amount` `(Type : Integer || Flag : Required)` : The no. of permissions to return. Use `-1` to return all.
> - `Separator` `(Type : String || Flag : Required)` : A text/character which is used for separating each permission.

Reviewer: ```suggestion
- `User ID` `(Type: Snowflake || Flag: Required)`: The user to get permissions for.
- `Return amount` `(Type: Integer || Flag: Required)`: The number of permissions to return. Use `-1` to return all.
- `Separator` `(Type: String || Flag: Required)`: The text to use for separating each permission.
```
